### PR TITLE
Initial blog post version

### PR DESCRIPTION
**Pull request by haroldtreen**
Here you go @yuliaju! Critique away! 👍

### CODE REVIEW

File: _posts/2016-10-08-recurse-center-week-2.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+categories: tech recurse
+---
+
+Another week of RC complete, another blog post!
+
+## Overall
+
+I've moved into a room at [Founder House](https://www.founder.house/). It's been a week there and I'm feeling settled in.
+
+During the week I was reflecting on my feelings. A lot of stuff are just things that take time to manifest.
```
Comment (yuliaju) [inline _posts/2016-10-08-recurse-center-week-2.md]: i'm confused about the meaning of the 2nd sentence. are you referring to your thoughts or making a general statement?

File: _posts/2016-10-08-recurse-center-week-2.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+#### I feel more relaxed.
+
+Having my own space has been awesome.
+
+I can grocery shop. I can cook. No more worrying about buying/storing food I won't be able to finish.  
+I can banter with housemates or sit in silence.  
+...I can do what I want.  
+
+It's not that I didn't have freedom before... I just restricted myself. No more of that!
```
Comment (yuliaju) [inline _posts/2016-10-08-recurse-center-week-2.md:28]: yay! :)

File: _posts/2016-10-08-recurse-center-week-2.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+#### I've felt slow.
+
+The features I worked on this week didn't seem *that* ambitious. I know exactly how I want everything to work... I just need to do it.
+
+Yet by the end of the week I was only about 1/2 way there. A lot of time went into refactoring/testing old components.
+
+It got me wondering if I'm being too much of a perfectionist. Maybe if I wasn't worrying about backwards compatibility, testing and architecture - it would be done already.
+
+At the same time I'm at RC to be a better programmer. I'm finding the more I obsess over decisions, the less I worry about them when they come up in the future. Experience cures indecisiveness.
```
Comment (yuliaju) [inline _posts/2016-10-08-recurse-center-week-2.md]: how are you currently thinking about reconciling your perfectionist tendencies vs. desire to become a better programmer?

**Review by yuliaju — COMMENTED** [review]
this is concise and informative! i don't have any major criticisms.

my only feedback: depending on what your goal is for writing the blog post, it could be good to give more detail on your day-by-day activities. it seems like your current goal is to summarize your wellness between weeks, but it could be helpful to add more information to see/record how certain activities (and occurrences that are currently too minor) affect your wellness, and even to detect trends or patterns. 

however, adding more information would detract from the conciseness, so putting it into a separate post or into a private journal could be another way of accomplishing the same goal. sending a daily overview to the facilitators is another solution (rose has explicitly said that she would be ok with this, haven't asked james yet)

File: _posts/2016-10-08-recurse-center-week-2.md (hunk 1 above)
Comment (haroldtreen) [reply _posts/2016-10-08-recurse-center-week-2.md]: There's defs some grammar issues going on there. I meant something like:

> The majority of the tension I feel are things that resolve over time.

Will clarify 👍

File: _posts/2016-10-08-recurse-center-week-2.md (hunk 3 above)
Comment (haroldtreen) [reply _posts/2016-10-08-recurse-center-week-2.md]: Good question! I think I'm just going to let myself be a perfectionist :P.

**Comment by haroldtreen** [comment]
Yay! Thanks @yuliaju :D

I agree some more day-to-day details could be informative. But would also agree that the point is more to summarize then dive into micro-details of activities. 

I do in fact have a log of day to day activities. It's hard to track stuff super in detail without having that become a time sink.

Added some changes and feeling good. Thanks for your help! 🎉

**Pull request merged**